### DISCUSSION TITLE
边怎么设置默认值

### DISCUSSION THREAD

**Discussion by minliuhua** [didi/LogicFlow#1374] [Q&A]
## 问题描述
我像在连接完边后，给他默认值，这个怎么操作

**Comment by wumail** [comment]
> ## 问题描述
> 我像在连接完边后，给他默认值，这个怎么操作

没懂你想设置什么默认值🤷

**Reply by minliuhua** [reply]
就是text.value,边显示的文本

**Reply by wumail** [reply]
你加一个edge:add 的回调，在回调里给text.value赋值，或者通过edgeGenerator给默认值应该都可以

**Reply by minliuhua** [reply]
十分感谢，按照您说的可以了，我的工作流快完工了
![image](https://github.com/didi/LogicFlow/assets/65475189/3092ca4a-6c1b-4c01-8255-5b386492418a)
